.gitlab-ci.yml

```diff
@@ -69,7 +69,7 @@ test:acceptance:
     - pip3 install --upgrade pip
     - cd tests
     - pip3 install -r requirements.txt
-    - python3 -m pytest -v --mender-version $MENDER_VERSION --mender-deb-version $MENDER_DEB_VERSION
+    - python3 -m pytest -v --mender-version "mender-"$MENDER_VERSION --mender-deb-version $MENDER_DEB_VERSION
```

Comment:
Technically speaking, if you hard-code here the `mender-` prefix this is this is now `INTEGRATION_VERSION`

Reply:
I mean it because today `integration` and `mender` are aligned, but you know this has not been the case in the past nor is guaranteed to be like this in the future.

Reply:
So you suggest we use the `release_tool` here?

Reply:
No, I suggest to either:
* rename the variable to `INTEGRATION_VERSION` and leave its value
* or set `MENDER_VERSION` to `mender-3.2.0` and revert this specific line.

Reply:
Do you see my point? We can discuss it f2f if I cannot make myself understandable here :sweat_smile:

Reply:
No, it makes sense.